test/test_prototype_transforms.py

```diff
@@ -152,3 +153,56 @@ def test_normalize(self, transform, input):
     )
     def test_random_resized_crop(self, transform, input):
         transform(input)
+
+
+class TestRandomHorizontalFlip:
+    def input_tensor(self, dtype: torch.dtype = torch.float32) -> torch.Tensor:
```

Comment:
@pmeier are we type-checking the tests?

If not, let's please remove all annotations.

Reply:
We are not. That would be madness even for me as `mypy` advocate. Still, do you want to enforce "no annotations" in the test suite? I agree they are not needed here, but for might be helpful for utility functions.

Reply:
I would prefer encouraging us not to use them. Type annotations hurt readability.
And when they're not being checked against, they can be wrong, which leads to confusion.

Reply:
I don't know if it hurts readability but it definitely hurts my laziness :D I agree with Nicolas that not having to deal with mypy on tests would be for the best. I would also be in favour of not putting any typing in the test suite.

Reply:
I agree with you that if tests are not type-checked, it's better to avoid type annotation.

So I removed it.